convert/src/vertex_to_bd/badgerdoc_format/plain_text_converter.py

```diff
@@ -0,0 +1,78 @@
+from typing import List
+
+from ..models import BadgerdocToken, Offset, Page, PageSize
+
+DEFAULT_PAGE_BORDER_OFFSET = 15
+
+
+def generate_chunks(text: str, n=70) -> List[str]:
+    return [text[i : i + n] for i in range(0, len(text), n)]
+
+
+class PlainTextToBadgerdocTokenConverter:
+    # TODO: calculate font width by font_height
+    def __init__(
+        self,
+        page_width: int = 595,
+        font_height: int = 11,
+        font_width: int = 7,
+        line_spacing: int = 2,
```

Comment:
Same as above for default values - leave a comment or remove from here

Reply:
fixed